tests/api/test_dash_acl_group.py

```diff
@@ -0,0 +1,50 @@
+
+from pprint import pprint
+
+import pytest
+
+@pytest.mark.dpu
```

Comment:
The CI/CD is failing because it's trying to run these DPU-specific TCs.
I believe the following two things should be done:
1. `-m "not dpu"` parameters into workflows whether these TCs are executed (not 100% sure I provided a valid key, should be checked)
2. To avoid accidental TCs execution without `-m "not dpu"`, we should add a fixture that validates target device type. Something like this:
```
@pytest.fixture(scope="module", autouse=True)
def skip_all(testbed_instance):
    testbed = testbed_instance
    if testbed is not None and len(testbed.dpu) != 1:
        pytest.skip("invalid for \"{}\" testbed".format(testbed.name))
```

Reply:
Added a fixture to test scripts